src/Apps/FeatureAKG/Components/FeaturedArticles.tsx

```diff
@@ -47,6 +44,10 @@ const FeaturedArticles: React.FC<FeaturedArticlesProps> = props => {
     })
   }
 
+  if (articles.length === 0) {
```

Comment:
We should keep the check against `null` articles, that was deliberate.

Reply:
It'd never be `null` though. If you look above the object is being spread into a new array (which is admittedly weird in itself).

Reply:
I guess so that it can mutate it with shift (why do that though).
(That object itself is a non-null array according to the typing at least)

Reply:
In that case, all good 👍